website/content/en/highlights/2023-09-26-0-33-0-upgrade-guide.md

```diff
@@ -34,9 +40,23 @@ with the other Datadog sinks, which use the `endpoint` as a base URL that the AP
 With this release, the `datadog_logs` sink's behavior is now consistent with the other
 Datadog sinks for the `endpoint` setting.
 
+#### Disable OpenSSL legacy provider by default {#openssl-legacy-provider}
+
+Vector upgraded the version of OpenSSL that it statically compiles in to v3.1.x in the 0.32.0 release.
+Following our deprecation policy, v0.33.0 now disables the legacy OpenSSL provider by default. It can be
+enabled through the [OpenSSL configuration options](https://www.openssl.org/docs/manmaster/man5/config.html)
```

Comment:
I know is tempting to be always "up-to-date", but I think is better practice to reference documentation for the specific version of OpenSSL currently being used in Vector (`3.1` instead of `master`):

<https://www.openssl.org/docs/man3.1/man5/config.html>

Otherwise, documentation can get out of sync when upstream releases new versions and Vector is not updated yet.

Reply:
Good catch.